.github/workflows/update-create-plugin.yml

```diff
@@ -13,4 +13,5 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: grafana/plugin-actions/create-plugin-update@main
+      - uses: grafana/plugin-actions/create-plugin-update@create-plugin-update/v2.0.1
```

Comment:
This workflow now requires a token.

Reply:
https://github.com/grafana/grafana-csv-datasource/pull/375/changes/28a57f74a11232f7d5caccc24556291cb863f572